Pod/Classes/LayoutConvenience.swift

```diff
@@ -57,3 +57,17 @@ public extension UIView {
         return container
     }
 }
+
+@available(iOS 9.0, *)
+public extension UIStackView {
+    
+    public func addArrangedSubviews(_ subviews: UIView...) {
+        addArrangedSubviews(subviews)
+    }
+    
+    public func addArrangedSubviews(_ subviews: [UIView]) {
```

Comment:
did you have a use case in your PDP code to pass an array at all? I guess it can't hurt, I just am not sure this will ever get used